egs/csj/s5/local/chain/tuning/run_tdnn_7k.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
+
+
+set -e
+
+# configs for 'chain'
+affix=
+stage=8
+train_stage=-10
+get_egs_stage=-10
+speed_perturb=false
+dir=exp/chain/tdnn_csj  # Note: _sp will get added to this if $speed_perturb == true.
+decode_iter=
+decode_nj=50
+
+# training options
+num_epochs=4
+initial_effective_lrate=0.001
+final_effective_lrate=0.0001
+leftmost_questions_truncate=-1
+max_param_change=2.0
+final_layer_normalize_target=0.5
+num_jobs_initial=1
+num_jobs_final=1
```

Comment:
... I guess you ran this somewhere where you had only one GPU?  If possible I'd increase num-jobs-{initial,final} to 3 and 10.  But you'll have to install GridEngine to run this. 
This will make it easier for others who have grids to reproduce your results quickly and reproducibly.